### ISSUE TITLE
How to iterate on all verb conjugations

### ISSUE THREAD

**Issue by poolebu** [SekouD/mlconjug#78]
* MLConjug version: 3.3.2
* Python version:  3.7.1
* Operating System: Windows / Anaconda

### Description

Hello,

This is more of a question than an issue. I would like to know if there is a general way to print all verb conjugations for a given language.

For example, for the input of: `lang=en` `verb=jump`, loop over `default_conjugator.conjugate(verb)` and print all tenses and conjugations, so users can get a quick idea on how to use a specific verb.

I see that  `default_conjugator.conjugate(verb)` returns a verb class specific for the language.

I'm kind of new to python, so a bit of guidance for this case would be very helpful to implement this awesome library with our project. 

Thank You!

**Comment by SekouD** [comment]
Hi @poolebu.

At the moment there is no simple way to iterate over all conjugation of a verb apart from doing:
for key, val in verb.items()

But this is a nice idea and I will implement a new method of the Verb class called Verb.iterate() that will return a list of all the conjugated forms in a canonical order.

I will implement it during the week and release a new version of mlconjug before the weekend.

Thanks for using mlconjug and for providing great feedback.

Cheers.

SekouD.

**Comment by poolebu** [comment]
Wow @SekouD thank you! I tried to use

```
verb = conjug_info=default_conjugator.conjugate(verb)
for key, val in verb.items()
```
But got an error, that verb has not items property.

Then I tried something like this but got stuck at the next level:

```
for tense in verb.conjug_info:
   print(tense)
   for x in tense.??
```
In short, I found `indicative` but not able to get `indicative past tense` for example

I will wait for your implementation. I have tested the software in a small group of users, and the value of getting the conjugations is huge to learn verbs in a new language. This is very exciting. I'll test as soon as it's ready. Thank you again for writing this.

**Comment by SekouD** [comment]
Thanks for the feedback @poolebu!

I will try to make the changes tonight and push the changes tomorrow.

I will let you know as soon as the new version has been pushed to pypi.

Thanks again!

SekouD

**Comment by SekouD** [comment]
Hi @poolebu , I just released version 3.4.0 of mlconjug.

It fixes a subtle bug when the verb's root never appears in its conjugated forms.

I also implemented your idea of being able to iterate over all conjugated forms.

To use this feature you can do the following:
```python
import mlconjug
default_conjugator = mlconjug.Conjugator(language='en')
test_verb = default_conjugator.conjugate("be")
all_conjugated_forms = test_verb.iterate()
print(all_conjugated_forms)
```
I uploaded [version 3.4.0](https://pypi.org/project/mlconjug/)

You can install the new version via `pip install mlconjug`

Thank you again for your suggestion, as I am sure it will be helpful to many users.

Do not hesitate to propose new features/suggestions or to report bugs when you find some.

Peace, love and have fun 👍 

SekouD.

**Comment by ernesto-butto** [comment]
Thank you!! It's working perfectly now! I'm figuring out how to present this in a digestible manner to the user. I'll let you know as soon as we launch a test version.
